### ISSUE TITLE
Add all 'aria' tags to form elements

### ISSUE THREAD

**Issue by esr360** [zendframework/zf2#7670]
Currently, for some strange reason, the only aria tags that Zend allows you to use on a form element are `aria-labelled-by` and `aria-described-by`, when the complete list of available attributes is [much longer](https://www.w3.org/TR/wai-aria/states_and_properties#state_prop_def), the particular one I need to use is `aria-label`.

These attributes are essential for providing usable experiences to disabled users and users with special needs. All attributes beginning with `aria-` should be whitelisted. Zend shouldn't be flagging these attributes as invalid - it's a shame that accessibility hasn't really been considered.

**Comment by froschdesign** [comment]
@esr360 
Please make your suggestion in the [zend-form issue tracker](https://github.com/zendframework/zend-form). Thanks!
